services/streams/streams/parsers.go

```diff
@@ -76,6 +76,17 @@ func (repetitive *Repetitive) ToMap() map[string]any {
 	return repetitiveMap
 }
 
+func (length *Length) ToMap() map[string]any {
+	lengthMap := make(map[string]any)
+	if length != nil {
```

Comment:
Can `length` be nil here? Because except if somewhere we explicitly set it to nil, I don't see how this is needed.